courses/tests/test_dredd.py

```diff
@@ -0,0 +1,40 @@
+"""
+Test driver for Dredd
+"""
+import os
+import subprocess
+
+from django.conf import settings
+from django.test import LiveServerTestCase
+
+
+class DreddTestCase(LiveServerTestCase):
+    "Tests Dredd related tests."
```

Comment:
Can you make this a triple quoted docstring? It's not a strict requirement but it's desired as a convention: https://www.python.org/dev/peps/pep-0257/